app/editor/src/features/admin/users/UserFilter.tsx

```diff
@@ -1,14 +1,25 @@
-import { IconButton } from 'components/form';
-import React from 'react';
+import { IconButton, OptionItem, Select } from 'components/form';
+import { IUserFilter, UserStatus, UserStatusName } from 'hooks';
+import React, { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import { Row, Text } from 'tno-core';
+import { useUsers } from 'store/hooks/admin';
+import { FieldSize, Text } from 'tno-core';
+import { Row } from 'tno-core/dist/components/flex';
 
-interface IUserFilterProps {
-  setGlobalFilter: (filterValue: any) => void;
-}
+interface IUserFilterProps {}
+
+export const UserFilter: React.FC<IUserFilterProps> = () => {
+  const [{ userFilter }, { storeFilter }] = useUsers();
+  const [filter, setFilter] = useState<IUserFilter>(userFilter);
+  const statusOptions = [
```

Comment:
I'm curious what's going on here?  Why the mix of number and string?  The API is currently configured to convert enum values to string when serializing.